packages/client/pages/ForgotPassword/ForgotPassword.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import { mustBeNotAuthenticated } from '~/store/auth'
+import { TextField, Button } from '@contentful/forma-36-react-components'
+
+class ForgotPasswordPage extends React.Component {
+    state = {
+        email: '',
+        errors: {},
+        isLoading: false,
+    }
+
+    submit = (submitEvent) => {
+        submitEvent.preventDefault()
+
+        this.setState({
+            isLoading: true,
+        })
+
+        Tensei.request
+            .post('forgot-password', {
+                email: this.state.email,
+            })
+            .then(() => {
+                this.setState({
+                    isLoading: false,
+                })
+
+                Tensei.library.Notification.error(
+                    `An email has been sent to the supplied email address. Follow the instruction in the email to reset your password`
+                )
+                this.props.history.push(Tensei.getPath('auth/login'))
+            })
+            .catch((error) => {
+                if (error?.response?.status === 422) {
```

Comment:
Please toast a message here if the message cannot be identified.

Reply:
I do not understand, more clarity please